### DISCUSSION TITLE
WatchOS support

### DISCUSSION THREAD

**Discussion by zRelux** [react-native-community/discussions-and-proposals#391] [Ideas]
Now this question stems from my lack of knowledge, but I remember reading somewhere that watchos was not supported from react-native because there was not JSC on it. Now that we have hermes on ios could it be viable to do something on watchOS how's it looking like, will there ever be a possibility?

**Comment by muhammadqazi** [comment]
I want to see also react native in watch os

**Comment by birminghamallen** [comment]
I three want to see react native on watch OS.
